CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v6.1.0 (upcoming)
+
+* Add `delete` to `ProfileDetail` view
```

Comment:
I'd call this backwards incompatible, as some developers might be perturbed to discover this new functionality available to their users, and may wish to protect against data loss. Archiving rather than deleting, for example, or making use of `on_delete`.